src/cpu/jit_avx512_common_convolution_winograd.cpp

```diff
@@ -922,8 +922,8 @@ void _jit_avx512_common_convolution_winograd_t<is_fwd>::_execute_data_W_S_G_D(
             last_slice_bias[oc] = bias(jcp.dimM / jcp.dimM_simd_block - 1, oc);
     }
 
-    PRAGMA_OMP(parallel) {
-        parallel_nd_in_omp(jcp.mb, jcp.dimK_nb_block, jcp.dimK_block,
```

Comment:
Do you have performance data that demonstrates changes in this commit do not negatively affect performance with OpenMP threading?

Reply:
No, we don't have performance data with OpenMP as we support only TBB. If you find any performance regressions, you could keep both versions using ifdefs. It would be a bit ugly, but it would avoid any potential issues.

Reply:
We cannot merge this then.

Reply:
Is this because you've found performance regressions with OpenMP?

Reply:
No, because we don't know if this change introduces regressions or not. So until we have performance data and are sure that there are no regressions, we cannot merge this.

Reply:
Sure, that makes sense. But unfortunately we don't have resources to provide our own performance data for OMP or other workloads. Could you please check this with your benchmark suite?